src/test/java/com/znsio/api/fetchPostsAndComments.feature

```diff
@@ -0,0 +1,17 @@
+@UserPosts @prod
+Feature: Fetch User Posts and Comments
+  Background:
+    * def userId = 1
+    * def testData = read('classpath:com/znsio/templates/userData.json')
+  @getPosts
+  Scenario: Get posts with specific User id
+    * def getPostsResponse = karate.call('classpath:com/znsio/templates/fetchPostAndComment.feature@t_getPost', {"userId": userId})
```

Comment:
'getPostsResponse' variable name need improvement. 

Or  you can use as below:

karate.call('classpath:com/znsio/templates/fetchPostAndComment.feature@t_getPost', {"userId": userId}).response

Reply:
renamed the variable..